BrainPortal/app/models/boutiques_portal_task.rb

```diff
@@ -622,4 +627,21 @@ def file_access_symbol
     @_file_access ||= (self.class.properties[:readonly_input_files].present? || self.tool_config.try(:inputs_readonly) ? :read : :write)
   end
 
+  private
+
+  # Prepare an array with revision information of
+  # all the Boutiques integrator modules used by the
+  # tools.
+  def boutiques_module_information #:nodoc:
+    descriptor = self.descriptor_for_final_task_list
+
+    integrator_modules = descriptor.custom['cbrain:integrator_modules'] || {}
+
+    return module_information = integrator_modules.map do |module_name, value|
```

Comment:
1. The `return module_information =` is unnecessary, since you're returning the value of the map anyway.
2. the 'value' variable is never used, so replace it with `_`